util/build_configs/cray-internal/chapel.spec.template

```diff
@@ -16,7 +16,7 @@ Name:    %{name}
 Version: %{version}
 Release: %{pkg_release}
 Prefix:  %{platform_prefix}
-License: Copyright 2021 Hewlett Packard Enterprise Development LP All Rights Reserved.
+License: Copyright 2022 Hewlett Packard Enterprise Development LP All Rights Reserved.
```

Comment:
@bradcray - I think it's also right for this one to only list 2022?

Reply:
Hmm, good question.  This one I'm even less familiar with, but checking its history, it looks like it's only ever given a single year at a time, so I think you're right (though I also am not convinced it would be wrong if it listed a range of years, like 2020-2022, since those are the years the copyright's been in HPE's name... that said, I'd probably stick to a single one).